src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -33,6 +33,11 @@ export class TaskListBuilder {
     const tasks: TaskListItem[] = []
     const externalId: string = claim.externalId
 
+    if (!ClaimFeatureToggles.areAdmissionsEnabled(claim)
+      && (draft.isResponsePartiallyAdmitted() || draft.isResponseFullyAdmitted())) {
+      delete draft.response.type
+    }
+
```

Comment:
Logic is fine but it feels it doesn't belong there...Please get another opinion @robertparkinson @andrewtaylorkainos @sabahirfan

Reply:
It is only temporary until admissions is switched on. I have tried a couple areas to add this but this one seems appropriate at the moment